socorro/cron/jobs/verify_processed.py

```diff
@@ -0,0 +1,151 @@
+        crash_id = raw_crash_key.split('/')[-1]
+
+        processed_crash_key = bucket.get_key(processed_crash_template % crash_id)
+        if processed_crash_key is None:
+            missing.append(crash_id)
+
+    return missing
+
+
+@as_backfill_cron_app
+class VerifyProcessedCronApp(BaseCronApp):
+    app_name = 'verifyprocessed'
+    app_description = 'verifies that incoming crash reports were processed'
+    app_version = '1.0'
+
+    required_config = Namespace()
+    required_config.add_option(
+        'crashstorage_class',
+        default='socorro.external.boto.crashstorage.BotoS3CrashStorage',
+        doc='S3 crash storage class',
+        from_string_converter=class_converter,
+        reference_value_from='resource.boto'
+    )
+    required_config.add_option(
+        'database_class',
+        default='socorro.external.postgresql.connection_context.ConnectionContext',
+        from_string_converter=class_converter,
+        reference_value_from='resource.postgresql'
+    )
+    required_config.add_option(
+        'date',
+        default='backfill',
+        doc=(
+            'The YYYYMMDD date. By default, this will use the backfill date from '
+            'the db.'
+        )
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.database = self.config.database_class(self.config)
+        self.crashstorage = self.config.crashstorage_class(self.config)
+
+    def get_entropy(self):
+        """Generate all entropy combinations."""
+        chars = '0123456789abcdef'
+        for x in chars:
+            for y in chars:
+                for z in chars:
+                    yield x + y + z
+
+    def find_missing_multiprocessing(self, date):
+        connection_source = self.crashstorage.connection_source
+        bucket_name = connection_source.config.bucket_name
+        boto_conn = connection_source._connect()
+
+        num_workers = 5
+        chunksize = 1
+
+        check_crashids_for_date = partial(
+            check_crashids,
+            boto_conn=boto_conn,
+            bucket_name=bucket_name,
+            date=date
+        )
+
+        pool = multiprocessing.Pool(num_workers)
+        missing = more_itertools.flatten(
+            pool.map(
+                check_crashids_for_date,
+                self.get_entropy(),
+                chunksize=chunksize
+            )
+        )
+        return list(missing)
+
+    def handle_missing(self, date, missing):
+        """Report crash ids for missing processed crashes."""
+        metrics.gauge('missing_processed', len(missing))
+        if missing:
+            for crash_id in missing:
+                self.logger.info('Missing: %s', crash_id)
+
+                with transaction_context(self.database) as conn:
+                    sql = """
+                    INSERT INTO crashstats_missingprocessedcrashes (crash_id, created)
+                    VALUES (%s, current_timestamp)
+                    """
```

Comment:
For now, it's dumping the crash ids in a db table. I'll spin off a new bug to create a Django admin action for reprocessing items in the table and one for checking the items in the table. I think that gives us everything we need to follow up on things.